apps/fluent-tester/src/FluentTester/TestComponents/Checkbox/CheckboxTest.tsx

```diff
@@ -103,7 +103,7 @@ const tokenCheckbox: React.FunctionComponent<{}> = () => {
 
   const theme = useTheme();
   const textBoxBorderStyle = {
-    borderColor: theme.colors.inputBorder,
+    borderColor: theme.colors.inputBorder as ColorValue,
```

Comment:
What actually is the TS type and the runtime type of `inputBorder`?

Reply:
It's ColorValue (which with this PR, will be the same type as ColorValue in React Native). You can find it defined here: https://github.com/microsoft/fluentui-react-native/blob/3c0f5496f0daeb748c6b2daac8d0465a1d58a672/packages/theming/theme-types/src/Color.types.ts#L249

At runtime, at the moment on all platforms except MacOS this is a string. On MacOS this will be a PlatformColor.

Reply:
I removed this cast and cast the entire object into a TextStyle isntead.

Reply:
Turns out it helps to do full builds to test changes. It seems that with full builds and being better about declaring types for objects that we create. we're able to avoid casting, so this is not an issue anymore.

Reply:
Well I spoke too soon. I can remove casts, but I still need to have the change done in DT in order to get this to compile fully. I have a PR open here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53368

Reply:
Okay it builds! I just had to use ColorValue from react-native directly and update all of the dependencies accordingly, instead of using ColorValue exported from theme-types (even though it's just the react-native one underneath)